src/ghostfat.c

```diff
@@ -339,16 +315,16 @@ void uf2_read_block (uint32_t block_no, uint8_t *data)
   memset(data, 0, BPB_SECTOR_SIZE);
   uint32_t sectionRelativeSector = block_no;
 
-  if ( block_no == 0 ) // aka requesting boot block
+  if ( block_no == 0 )
   {
-    // Requested boot block
+    // Request Boot block
```

Comment:
The old word "Requested" was past-tense, and referred to the command that was received.

Changing these to "Request" (present-tense) doesn't make sense, because the code is not making a request.

If you want to change the tense from past to present, perhaps you meant "create" or "generate"?